main.cpp

```diff
@@ -59,84 +60,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: TV
+//  action 1: TV On
+tv.on();
+//  action 2: TV Off
+tv.off();
+//  action 3: TV changes channel
+tv.changeChannel();
  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: printer
+//  action 1: printer prints
+printer.print();
+//  action 2: printer runs out of tint
+printer.outOfTint();
+//  action 3: printer turns on wi-fi connection
+printer.wifiConnectionOn();
  
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: person
+//  action 1: person seats
+person.seat();
+//  action 2: person jumps
+person.jump();
+//  action 3: person shouts
+person.shouts();
 
-//  action 3:
- 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: dog 
+//  action 1: dog barks
+dog.bark();
+//  action 2: dog smells poop
+dog.smellPoop();
+//  action 3: dog gets lost
+dog.getLost();
  
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: car
+//  action 1: car crashes
+car.crash();
+//  action 2: car breaks
+car.break();
```

Comment:
`break` is a reserved word, which is why it's red here.  Do you mean "break" like "break down", or "brake", like "slow down"?

- [ ] pick a different action and revise.